airbyte-cdk/python/airbyte_cdk/sources/file_based/config/file_based_stream_config.py

```diff
@@ -66,31 +66,25 @@ class FileBasedStreamConfig(BaseModel):
     file_type: str
     globs: Optional[List[str]]
     validation_policy: Union[str, Any]
-    validation_policies: Dict[str, Any]
     catalog_schema: Optional[ConfiguredAirbyteCatalog]
     input_schema: Optional[Dict[str, Any]]
     primary_key: PrimaryKeyType
     max_history_size: Optional[int]
     days_to_sync_if_history_is_full: Optional[int]
     format: Optional[Mapping[str, CsvFormat]]  # this will eventually be a Union once we have more than one format type
 
+    @validator("file_type", pre=True)
+    def validate_file_type(cls, v):
+        if v not in VALID_FILE_TYPES:
+            raise ValueError(f"Format filetype {v} is not a supported file type")
+        return v
+
     @validator("format", pre=True)
     def transform_format(cls, v):
         if isinstance(v, Mapping):
             file_type = v.get("filetype", "")
-            if file_type.casefold() not in VALID_FILE_TYPES:
-                raise ValueError(f"Format filetype {file_type} is not a supported file type")
-            return {file_type: {key: val for key, val in v.items()}}
+            if file_type:
```

Comment:
fixed a small bug. The new way we present formats would always fail since the empty string is not a valid file type. We should only perform the check if we know this is the legacy format, delineated by the `filetype` being present